amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/v0/CustomDefinitionsAPI.java

```diff
@@ -41,6 +38,8 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
+import static org.testng.AssertJUnit.*;
```

Comment:
We decided to standardise on imports ordered with static first, and java.* before org.*.  Also we turned off wildcard imports. There's an IDE settings file in the root of the AGS project.